src/styles/components/card.scss

```diff
@@ -19,7 +19,7 @@ $block: ns(card);
     $tn-card-media-width: 75px !default;
     $tn-card-color-border: tn-color(neutral,2) !default;
     $tn-card-color-disabled: tn-color(text,3) !default;
-    $tn-card-background-color: tn-color(inverse) !default;
+    $tn-card-background-color: tn-color(text-inverse) !default;
```

Comment:
@joseegm The most important part of theming is that colors be identified by use not actual color. Beyond that, the design system is owned by design. The library architecture is designed to absorb changes but I agree that we need to narrow down the system as soon as possible.

I'm happy with system, I'll leave updates to you guys. Just please try to keep the palette limited. The larger it grows the harder it is to understand the use and theming becomes more difficult. 

Thanks.